### PR TITLE
Update changelog for v7.0.0-beta2

### PR DESCRIPTION
**Pull request by marefr**
Did some updates of PR titles in the PR's before generating. Anything missing?

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,12 +49,64 @@ Grafana 7.0 comes with a new table panel (and deprecates the old one). This new
...
+- **Dashboard**: A global minimum dashboard refresh interval is now enforced and defaults to 5 seconds.
+- **@grafana/ui**: Forms migration notice, see [@grafana/ui changelog](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/CHANGELOG.md)
+- **Interval calculation**: There is now a new option `Max data points` that controls the auto interval `$__interval` calculation. Interval was previously calculated by dividing the panel width by the time range. With the new max data points option it is now easy to set `$__interval` to a dynamic value that is time range agnostic. For example if you set `Max data points` to 10 Grafana will dyanmically set `$__interval` by dividing the current time range by 10.
+- **Datasource/Loki**: Support for [deprecated Loki endpoints](https://github.com/grafana/loki/blob/master/docs/api.md#lokis-http-api) has been removed.
+
+**Deprecation warnings**
+
+- Scripted dashboards is now deprecated. The feature is not removed but will be in a future release. We hope to address the underlying requirement of dynamic dashboards in a different way. [#24059](https://github.com/grafana/grafana/issues/24059)
```
Comment (dprokop) [inline CHANGELOG.md:71]:
```suggestion
- Scripted dashboards are now deprecated. The feature is not removed but will be in a future release. We hope to address the underlying requirement of dynamic dashboards in a different way. [#24059](https://github.com/grafana/grafana/issues/24059)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,12 +49,64 @@ Grafana 7.0 comes with a new table panel (and deprecates the old one). This new
...
+
+### Features / Enhancements
+
+* **CloudWatch**: Adds more examples to CloudWatch Logs cheatsheet. [#24288](https://github.com/grafana/grafana/pull/24288), [@kaydelaney](https://github.com/kaydelaney)
+* **Elasticsearch**: Changes terms min_doc_count default from 1 to 0. [#24204](https://github.com/grafana/grafana/pull/24204), [@Estrax](https://github.com/Estrax)
+* **LoginPage**: New design. [#23892](https://github.com/grafana/grafana/pull/23892), [@torkelo](https://github.com/torkelo)
```
Comment (dprokop) [inline CHANGELOG.md:77]:
```suggestion
* **Login Page**: New design. [#23892](https://github.com/grafana/grafana/pull/23892), [@torkelo](https://github.com/torkelo)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,12 +49,64 @@ Grafana 7.0 comes with a new table panel (and deprecates the old one). This new
...
+* **Logs**: Add log level Fatal. [#24185](https://github.com/grafana/grafana/pull/24185), [@davkal](https://github.com/davkal)
+* **Loki**: Show loki datasource stats in panel inspector. [#24190](https://github.com/grafana/grafana/pull/24190), [@davkal](https://github.com/davkal)
+* **Migration**: Dashboard links. [#23553](https://github.com/grafana/grafana/pull/23553), [@peterholmberg](https://github.com/peterholmberg)
+* **Plugins**: Require signing of external back-end plugins. [#24075](https://github.com/grafana/grafana/pull/24075), [@aknuds1](https://github.com/aknuds1)
+* **Prometheus**: Add off switch for metric/label name lookup. [#24034](https://github.com/grafana/grafana/pull/24034), [@s-h-a-d-o-w](https://github.com/s-h-a-d-o-w)
+* **Search**: Bring back open search by clicking dashboard name. [#24151](https://github.com/grafana/grafana/pull/24151), [@torkelo](https://github.com/torkelo)
+* **Tracing**: Header updates. [#24153](https://github.com/grafana/grafana/pull/24153), [@aocenas](https://github.com/aocenas)
+* **Transformations**: Improve timeseries support. [#23978](https://github.com/grafana/grafana/pull/23978), [@ryantxu](https://github.com/ryantxu)
```
Comment (dprokop) [inline CHANGELOG.md:85]:
```suggestion
* **Transformations**: Improve time series support. [#23978](https://github.com/grafana/grafana/pull/23978), [@ryantxu](https://github.com/ryantxu)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -48,12 +49,64 @@ Grafana 7.0 comes with a new table panel (and deprecates the old one). This new
...
+* **Field options**: show field name when title option config is empty. [#24335](https://github.com/grafana/grafana/pull/24335), [@dprokop](https://github.com/dprokop)
+* **Graph**: Fixed graph tooltip getting stuck / not being cleared when leaving dashboard. [#24162](https://github.com/grafana/grafana/pull/24162), [@torkelo](https://github.com/torkelo)
+* **Graph**: Fixed issue with x-axis labels showing "MM/DD" after viewing dashboard with pie chart. [#24341](https://github.com/grafana/grafana/pull/24341), [@mckn](https://github.com/mckn)
+* **Jaeger**: Fix how label is created in cascader. [#24164](https://github.com/grafana/grafana/pull/24164), [@aocenas](https://github.com/aocenas)
+* **Loki**: Fix label matcher for log metrics queries. [#24238](https://github.com/grafana/grafana/pull/24238), [@ivanahuckova](https://github.com/ivanahuckova)
+* **PanelInspector**: hides Query tab for plugins without Query ability. [#24216](https://github.com/grafana/grafana/pull/24216), [@hugohaggmark](https://github.com/hugohaggmark)
```
Comment (dprokop) [inline CHANGELOG.md:104]:
```suggestion
* **Panel inspect**: hides Query tab for plugins without Query ability. [#24216](https://github.com/grafana/grafana/pull/24216), [@hugohaggmark](https://github.com/hugohaggmark)
```

**Review by dprokop — APPROVED** [review]

**Review by aknuds1 — APPROVED** [review]
LGTM once dprokop's suggestion is merged :)

**Review by aocenas — APPROVED** [review]

**Review by xlson — COMMENTED** [review]
There is one Enterprise PR that should go in here as well. 

`Reporting: update report schedule as soon as a reported is updated (Enterprise)`

**Comment by marefr** [comment]
@xlson I'll fix :+1:

**Review by daniellee — APPROVED** [review]
LGTM

**Comment by aknuds1** [comment]
@marefr There's now a merge conflict :/

**Pull request merged**